src/ocnicepost.fd/init_mod.F90

```diff
@@ -78,12 +78,11 @@ subroutine readnml
     close (iounit)
     nxt = srcdims(1); nyt = srcdims(2)
     nxr = dstdims(1); nyr = dstdims(2)
```

Comment:
Is nlevs (the number of vertical levels for the ocean model) no longer needed or used?

Reply:
it is needed and used.  
It is just not being provided via the namelist.  It is being read in from the `ocean.nc` file in [ocnicepost.F90](https://github.com/NOAA-EMC/gfs-utils/pull/45/files#diff-c64691f1cb9756e5e066f35c8a4c3cd3610b87420d39df627ce4bc56474c3e40)

Reply:
I see. Thanks.